Sources/PackageDescription/PackageDescriptionSerialization.swift

```diff
@@ -76,6 +86,18 @@ extension Package.Dependency.Requirement: Encodable {
         case localPackage
     }
 
+    /// Encodes this value into the given encoder.
+    ///
+    /// If the value fails to encode anything, `encoder` will encode an empty
+    /// keyed container in its place.
+    ///
+    /// This function throws an error if any values are invalid for the given
+    /// encoder's format.
+    ///
+    ///  - Note: This documentation comment was inherited from `Encodable`.
+    ///
+    /// - Parameters:
+    ///   - encoder: The encoder to write data to.
```

Comment:
If the documentation is inherited from the default implementation of `encode`, then imo it shouldn't be included.

Reply:
The idea is to have some documentation that makes sense. If the inherited doc does not make sense, that's a problem, and I have written replacement docs in the few cases of protocol conformance docs that I found. Is that the case here too?

Reply:
Given that this extension is deprecated, and it's an internal implementation detail, I will simplify the documentation for this function.